src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -140,6 +140,61 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
         }
     }
 
+    override fun createPersonOversiktStatusIfMissing(personident: PersonIdent): Boolean {
```

Comment:
Er det `existing.isEmpty()` som bestemmer hva som blir returnert her? Ville nok vært lettere å lese om sjekk på person eksisterer og create var to ulike funksjoner.

Reply:
Ja, poenget er å opprette forekomsten dersom den ikke finnes fra før. 

Forsøker å dele opp (men det er jo et poeng å bruke samme connection)